projects/core/src/state/utils/entity/entity.action.ts

```diff
@@ -5,11 +5,11 @@ export const ENTITY_REMOVE_ALL_ACTION = '[ENTITY] REMOVE ALL';
 
 export interface EntityMeta {
   entityType: string;
-  entityId: string | string[];
+  entityId?: string | string[];
```

Comment:
Why does this field ( and further down the road function param) has to be now optional?

Reply:
No. It should be optional. Because in `entity.reducer.ts` there is check `action.meta.entityId !== undefined` (line 26)